app/views/users/_form.html.erb

```diff
@@ -26,7 +26,7 @@
     <% else %>
       <%= form.label :toggle, "Change this user into an admin?" %>
     <% end %>
-    <%= check_box_tag(:toggle) %>
+    <%= radio_button_tag(:toggle, true) %>
```

Comment:
Hmm, what I mean is not like this tbh lol, 

the problem is if you only have 1 radio button, then after you click it then you cannot unselect, what i mean was something like you have 2 options (admin / normal user) marked with radio button, then the superuser can just choose based on those 2 options.

Reply:
Ok. Fixed.